### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by nyouness** [danyalutsevich/FireFly#74]
Hii i been using this amazing and underrated project and i love it and my only issue is it does not support Tv Show completely ( does not show or load in my watchlist/Liked , covers not showing ) please add full support to series pleaaaase pleasse something like this :
![Menu](https://github.com/danyalutsevich/FireFly/assets/117232906/f1fb3c7f-9778-492a-b2da-1475fb21e589)
![Episodes](https://github.com/danyalutsevich/FireFly/assets/117232906/3c94faf9-932b-4922-a787-eb5a2afb993d)

and thanks a looooooooot for this project and in advance !

**Comment by danyalutsevich** [comment]
Hello there! Thank you for your kind words and feedback about the project; we're thrilled to hear that you're enjoying it!
We will keep in mind your suggestion and implement it when someone from our team will have free time.

I think it would be hard to find single episode torrents. Most torrents include an entire season or all shows.

Thank you for your kind words

**Comment by nyouness** [comment]
and here is the config for Torrentio addon [https://torrentio.strem.fun/configure](url)  its the best torrent addon you can find in any BitTorrent software

**Comment by danyalutsevich** [comment]
yeah we using this addon
thank you

Im not guarantee you that this feachure will be implemented in the nearest 2 month cause im a little busy at the moment
But who knows maybe i'll get inspired to make it

**Comment by danyalutsevich** [comment]
By the way dude how did you find our website?

**Comment by nyouness** [comment]
I love open source projects and i like to search for projects/underrated projects in github , i was looking for a stremio/popcorntime BitTorrent software like and i found your project by the tags

**Comment by nyouness** [comment]
Do you guys have any attention to make your website into a BitTorrent software something like Stremio or Bee-desktop instead of just a website ? i think it will be cool

**Comment by danyalutsevich** [comment]
I'd love and i tried to play the movies right in the browser without downloading webtorrent.
But for now it is impossible (it is possible but the speed is too bad)
Something like this:
 https://ferrolho.github.io/magnet-player/
 https://webtorrent.io/

Web browser does not allow to create sockets, only websockets.
And as i understand webtorrent.io can only communicate with the same browsers but not the original torrent clients

**Comment by nyouness** [comment]
Yeah i know both of them and i never used webtorrent like to stream torrents in browser i like BitTorrents software .
I mean make a desktop BitTorrent software and implement all the things in Firefly + my feature request , maybe fork a BitTorrent software like popcorntime or webtorrent desktop or Bee-desktop or anyone that support streaming torrents and enhance it .

**Comment by nyouness** [comment]
Hii , i just want to ask is there any news ?

**Comment by danyalutsevich** [comment]
No news for now. I remember about you feature request.Will implement it when i'll be free

**Comment by danyalutsevich** [comment]
Hey i still remember about reachure you requested but im busy with other project for now
P.S. fireflyratings.com domain expired
you can access the website by this link https://firefly-49286.web.app/
